src/app/modules/main/chat_section/chat_content/io_interface.nim

```diff
@@ -3,7 +3,7 @@ import NimQml
 import ../item as chat_item
 import ../../../../../app_service/service/message/dto/pinned_message
 import ../../../../../app_service/service/chat/dto/chat
-
+import ../../../../../app_service/service/message/dto/message
```

Comment:
@jrainville you could go without `../../../../../`, actually you can remove those prefixes from every line here, but even if not, we can live with that. ;)